From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Mon, 12 Feb 2024 09:39:42 +0100
Subject: docs/conf.py: Use sphinx_rtd_theme instead

The used theme by upstream (pydata_sphinx_theme) isn't packaged in
Debian, falling back than to the classical RTD theme.

Forwarded: not-needed
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index fa5647a..337135c 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -84,7 +84,7 @@ todo_include_todos = False
 # a list of builtin themes.
 #
 
-html_theme = "pydata_sphinx_theme"
+html_theme = "sphinx_rtd_theme"
 html_logo = "_static/logo_short.png"
 
 # Theme options are theme-specific and customize the look and feel of a theme
